### DISCUSSION TITLE
I can't upload more than 10 files with media library pro

### DISCUSSION THREAD

**Discussion by hnooz** [spatie/laravel-medialibrary#3039] [Q&A]
Hi there 
I'm trying to upload more than 10 images in one time but i got 429

![Screenshot from 2022-09-22 09-38-03](https://user-images.githubusercontent.com/44949769/191687000-2be23517-4b6a-4aa8-a959-34321f9835e6.png)

**Comment by parallels999** [comment]
You have to buy the **PRO+** version
